src/Microsoft.FeatureManagement/IContextualFeatureVariantAssigner.cs

```diff
@@ -7,18 +7,18 @@
 namespace Microsoft.FeatureManagement
 {
     /// <summary>
-    /// Provides a method to assign a variant of a feature to be used based off of custom conditions.
+    /// Provides a method to assign a variant of a dynamic feature to be used based off of custom conditions.
     /// </summary>
     /// <typeparam name="TContext">A custom type that the assigner requires to perform assignment</typeparam>
     public interface IContextualFeatureVariantAssigner<TContext> : IFeatureVariantAssignerMetadata
     {
         /// <summary>
-        /// Assign a variant of a feature to be used based off of customized criteria.
+        /// Assign a variant of a dynamic feature to be used based off of customized criteria.
         /// </summary>
-        /// <param name="variantAssignmentContext">A variant assignment context that contains information needed to assign a variant for a feature.</param>
-        /// <param name="appContext">A context defined by the application that is passed in to the feature management system to provide contextual information for assigning a variant of a feature.</param>
+        /// <param name="variantAssignmentContext">A variant assignment context that contains information needed to assign a variant for a dynamic feature.</param>
+        /// <param name="appContext">A context defined by the application that is passed in to the feature management system to provide contextual information for assigning a variant of a dynamic feature.</param>
         /// <param name="cancellationToken">The cancellation token to cancel the operation.</param>
-        /// <returns>The variant that should be assigned for a given feature.</returns>
+        /// <returns>The variant that should be assigned for a given dynamic feature.</returns>
         ValueTask<FeatureVariant> AssignVariantAsync(FeatureVariantAssignmentContext variantAssignmentContext, TContext appContext, CancellationToken cancellationToken);
```

Comment:
>cancellationToken [](http://example.com/codeflow?start=150&length=17)

Should we default the cancellationToken?

Looks like we don't have cancellationToken in I*FeatureFilter, should we add it?

Reply:
> Looks like we don't have cancellationToken in I*FeatureFilter, should we add it?

cancellation tokens on existing interfaces were added in PRs checked in to the main branch.
* https://github.com/microsoft/FeatureManagement-Dotnet/pull/131

For that reason I didn't touch them here. This PR is going into "feature/v3" branch and when "feature/v3" merges into main those conflicts will be resolved.

Reply:
Circling back to this, there were some missing default cancellation tokens. Thanks. I've added them.